test/powershell/Modules/Microsoft.PowerShell.Management/Start-Process.Tests.ps1

```diff
@@ -69,5 +68,17 @@ Describe "Start-Process" -Tags @("CI","SLOW") {
 	$dirEntry.Length | Should BeGreaterThan 0
     }
 
+    It "Should give an error when Verb parameter is used" -Skip:(-not $IsCoreClr) {
+        try 
+        {
+            Start-Process -Verb runas -FilePath $pingCommand -ArgumentList $pingParam -ErrorAction Stop
+            throw "No Exception!"
+        }
+        catch
+        {
+            $_.FullyQualifiedErrorId | Should be "NotSupportedException,Microsoft.PowerShell.Commands.StartProcessCommand"
```

Comment:
>                            else if (error == 0x424)
                            {
                                // The API 'CreateProcessWithLogonW' depends on the 'Secondary Logon' service, but the component 'Microsoft-Windows-SecondaryLogonService' 
                                // is not installed in OneCoreUAP. We will get error code 0x424 when the service is not available.
                                message = StringUtil.Format(ProcessResources.ParameterNotSupported, "-Credential", "Start-Process");
                                er = new ErrorRecord(new NotSupportedException(message), "NotSupportedException", ErrorCategory.NotInstalled, null);
                            }

I find `NotSupportedException` may be thrown from Start-Process by other code path too. Can you please check the error message to see if it contains `-Verb`?